### ISSUE TITLE
warping to anat has weird result

### ISSUE THREAD

**Issue by jcohenadad** [neuropoly/spinalcordtoolbox#300]
data: sct_example_data

syntax:
~~~~
sct_propseg -i t2.nii.gz -t t2 -mesh -max-deformation 4 -init 130
sct_register_to_template -i t2.nii.gz -l labels.nii.gz -s t2_seg.nii.gz
sct_warp_template -d t2.nii.gz -w warp_template2anat.nii.gz
~~~~

Result:

![screen shot 2015-04-02 at 19 18 02](https://cloud.githubusercontent.com/assets/2482071/6975686/32ab8934-d96d-11e4-906c-1f564de7bbab.png)

Maybe this is related to the crop of the destination image.

**Comment by benjamindeleener** [comment]
Probably caused by an incorrect reference image when applying the warping field on the template. The slice circled is probably a remanent of the tempalte segmentation.

The issue is referenced here: #301

**Comment by benjamindeleener** [comment]
Should be fixed in c86f6616e05b429fc9d809e9c04da1bf498b755e
